### PR TITLE
Clean up unneed includes

### PR DESCRIPTION
**Pull request by crizzitello**
Clean up unneeded includes
Remove std::err and std::cout for Qt methods

### CODE REVIEW

**Review by JoelAtDeluxe — DISMISSED** [review]
Looks good.

As an aside, I wonder if we should keep around the CLI portion of the application (which currently does next-to-nothing). Originally, I think we might have wanted to support CLI-only type environments, but I suspect that's unlikely to be useful in practice.

**Review by jkennedyvz — DISMISSED** [review]

**Review by jrozner — APPROVED** [review]

File: src/forms/evidence/evidencemanager.cpp [hunk 1]
```diff
@@ -440,7 +439,7 @@ void EvidenceManager::refreshRow(int row) {
     setRowText(row, updatedData);
   }
   catch (QSqlError& e) {
-    std::cout << "Could not refresh table row: " << e.text().toStdString() << std::endl;
+    QTextStream(stdout) << "Could not refresh table row: " << e.text() << Qt::endl;
```
Comment (jrozner) [inline src/forms/evidence/evidencemanager.cpp]: should this be stderr?

File: src/forms/evidence/evidencemanager.cpp [hunk 2]
```diff
@@ -507,8 +506,8 @@ void EvidenceManager::onUploadComplete() {
       db->updateEvidenceError(errMessage, evidenceIDForRequest);
     }
     catch (QSqlError& e) {
-      std::cout << "Upload failed. Could not update internal database. Error: "
-                << e.text().toStdString() << std::endl;
+      QTextStream(stdout) << "Upload failed. Could not update internal database. Error: "
```
Comment (jrozner) [inline src/forms/evidence/evidencemanager.cpp]: stderr?

File: src/forms/getinfo/getinfo.cpp [hunk 3]
```diff
@@ -141,8 +141,8 @@ void GetInfo::onUploadComplete() {
       db->updateEvidenceError(errMessage, evidenceID);
     }
     catch (QSqlError& e) {
-      std::cout << "Upload failed. Could not update internal database. Error: "
-                << e.text().toStdString() << std::endl;
+      QTextStream(stdout) << "Upload failed. Could not update internal database. Error: "
```
Comment (jrozner) [inline src/forms/getinfo/getinfo.cpp]: stderr?

File: src/forms/getinfo/getinfo.cpp [hunk 4]
```diff
@@ -157,8 +157,8 @@ void GetInfo::onUploadComplete() {
       close();
     }
     catch (QSqlError& e) {
-      std::cout << "Upload successful. Could not update internal database. Error: "
-                << e.text().toStdString() << std::endl;
+      QTextStream(stdout) << "Upload successful. Could not update internal database. Error: "
```
Comment (jrozner) [inline src/forms/getinfo/getinfo.cpp]: stderr?

**Review by jrozner — CHANGES_REQUESTED** [review]
I didn't end up marking all of them but it seems like most of the output here are errors but they are going to stdout. Should these get updated to stderr?

**Comment by crizzitello** [comment]
Updated with more unneeded includes removed also using std err for most of the TextStreams.

File: src/components/error_view/errorview.h [hunk 5]
```diff
@@ -1,11 +1,8 @@
 #pragma once
 
-#include <QLabel>
-#include <QString>
-#include <QWidget>
-
+//#include <QLabel>
```
Comment (jrozner) [inline src/components/error_view/errorview.h]: Is this supposed to be commented out with the forward class declaration below?

File: src/helpers/netman.h [hunk 6]
```diff
@@ -7,16 +7,13 @@
...
-#include <string>
-#include <vector>
 
 #include "appconfig.h"
 #include "request_builder.h"
 #include "dtos/operation.h"
 #include "dtos/tag.h"
 #include "dtos/github_release.h"
-#include "helpers/file_helpers.h"
+//#include "helpers/file_helpers.h"
```
Comment (jrozner) [inline src/helpers/netman.h]: Is this just not needed?

File: src/traymanager.cpp [hunk 7]
```diff
@@ -252,7 +252,7 @@ void TrayManager::onScreenshotCaptured(const QString& path) {
     spawnGetInfoWindow(evidenceID);
   }
   catch (QSqlError& e) {
-    std::cout << "could not write to the database: " << e.text().toStdString() << std::endl;
+    QTextStream(stdout) << "could not write to the database: " << e.text() << Qt::endl;
```
Comment (jrozner) [inline src/traymanager.cpp]: Should this be stderr?

**Review by jrozner — COMMENTED** [review]
Changes look good

**Review by jrozner — APPROVED** [review]

**Pull request merged**